toast-tk-agent-api/src/main/java/io/toast/tk/agent/config/WebConfigProvider.java

```diff
@@ -0,0 +1,61 @@
+package io.toast.tk.agent.config;
+
+import java.io.File;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.io.File'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)